element.go

```diff
@@ -223,7 +234,7 @@ func (el *Element) DescribeE() (*proto.DOMNode, error) {
 
 // ShadowRootE returns the shadow root of this element
 func (el *Element) ShadowRootE() (*Element, error) {
-	node, err := el.DescribeE()
+	node, err := el.DescribeE(-1, true)
```

Comment:
To get the shadow element, we don't need the whole tree.

Reply:
use this  `(1, false)`

Reply:
yes